examples/hitl/rearrange_v2/rearrange_v2.py

```diff
@@ -251,16 +287,56 @@ def _get_camera_lookat_pos(self) -> mn.Vector3:
     def _is_user_idle_this_frame(self) -> bool:
         return not self.gui_input.get_any_input()
 
+    def _on_pick(self, e: UI.PickEventData):
+        self.ui_events.append(
+            {
+                "type": "pick",
+                "obj_handle": e.object_handle,
+                "obj_id": e.object_id,
+            }
+        )
+
+    def _on_place(self, e: UI.PlaceEventData):
+        self.ui_events.append(
+            {
+                "type": "place",
+                "obj_handle": e.object_handle,
+                "obj_id": e.object_id,
+                "receptacle_id": e.receptacle_id,
+            }
+        )
+
+    def _on_open(self, e: UI.OpenEventData):
+        self.ui_events.append(
+            {
+                "type": "open",
+                "obj_handle": e.object_handle,
+                "obj_id": e.object_id,
+            }
+        )
+
+    def _on_close(self, e: UI.CloseEventData):
+        self.ui_events.append(
+            {
+                "type": "close",
+                "obj_handle": e.object_handle,
+                "obj_id": e.object_id,
+            }
+        )
+
```

Comment:
This is irrelevant to PR, but do we need open/close for single-learn data collection?

Reply:
We don't need to do it right now. That will most likely change with the inclusion of object states.